### ISSUE TITLE
Submit to PyPI

### ISSUE THREAD

**Issue by romanvm** [xbmc/kodi-addon-submitter#9]
The title says it's all

**Comment by romanvm** [comment]
Done.

**Comment by dagwieers** [comment]
Yes please, or make releases from time to time so a change that breaks does not impact add-ons immediately.

**Comment by Razzeee** [comment]
You could also just refer to the commit hash instead of HEAD, but these would be better.

**Comment by dagwieers** [comment]
Well, my problem is not in this project. The problem is the GitHub action (v1.2) that always takes the latest master branch of this project. At the moment we cannot release add-ons because git is taking the wrong commits causing the wrong thing to be pushed to the Kodi repositories. Not sure why, but it seems everyone is likely affected.

**Comment by Razzeee** [comment]
Yes, I'v been having problems since this weekend too. Might not be the same ones.

But fixing each version of the action to a certain commithash here, would take out the guesswork. https://github.com/xbmc/action-kodi-addon-submitter/blob/master/Dockerfile#L7

**Comment by Razzeee** [comment]
Yeah, my problem is, that the pipeline fails before it even creates the PR
